### ISSUE TITLE
Automatic time-stamped backup at start & end

### ISSUE THREAD

**Issue by CaptainBloodz** [zombieFox/nightTab#222]
New user here, ~ a week with frefox + linux.
Very good to avoid eyes strain, imho.

Firefox had to be closed because of some multimedia issue.
Once restarted, user configuration was switched back to default.

I noticed configuration can be backuped,restored & cleared.
Unfirtunately, only after having being hit be the reset issue.

Would be nice to have a feature that backups at start and/or when closing to time-stamped files.

Thks 4 ur attention, interest & support.

**Comment by zombieFox** [comment]
Hi,

I'll try to help. Can you provide a few more details please?

- What browser (and the version) are you using?
- What OS (and the version) are you using?
- Do you have any other extensions installed which clear cache or cookies when closing the browser?

nightTab does indeed restore the backup on load. This feature is working as intended on the current version.
